spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -278,6 +278,11 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
         fun parse(expression: String): SpdxSingleLicenseExpression =
             SpdxExpression.parse(expression) as SpdxSingleLicenseExpression
     }
+
+    /**
+     * Return the string identifier of this license, without the optional exception.
```

Comment:
How about just saying "without any license exception" (and no comma before that)?

Reply:
Done.